public/npm

```diff
@@ -0,0 +1 @@
+../node_modules/
```

Comment:
Requesting delete, we should use JSDelivr instead of `node_modules`.

Reply:
Lmao we can serve from jsdelivr and still have this link for local development.

Reply:
IDK, feels too much like exposing internal details. I suppose I'd evaluate it at -.5, but I guess it doesn't really matter.